Assets/Scripts/Controllers/Menu/UIBuilderQuitWarning.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+public class UIBuilderQuitWarning : MonoBehaviour
+{
+    Button _yesButton;
+    Button _noButton;
+    VisualElement _root;
+    VisualElement _quitWarning;
+
+    
+    void OnEnable()
+    {
+        _root = GetComponent<UIDocument>().rootVisualElement;
+    	_quitWarning = GameObject.Find("QuitWarning").GetComponent<UIDocument>().rootVisualElement.Q<VisualElement>("Container");
+
+        _yesButton = _root.Q<Button>("YesButton");
+        _yesButton.clicked += OnYesButton;
+        _noButton = _root.Q<Button>("NoButton");
+        _noButton.clicked += OnNoButton;
+    }
+
+    void OnYesButton(){
+        Logger.Log("MenuController", "THE MENU IS DEAD!");
+		//Application.Quit();
```

Comment:
Why is this commented? The user can click the yes button but nothing would happen 😅

Reply:
This was Wouter, Application.Quit() doesnt work when testing in unity thats why it is logged to see if it is triggerd

Reply:
I'd say uncomment the line with the app quit so it works when in a built state 😁

Reply:
Already fixed on my branch 😅
Will add it here as well.